src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,56 @@
-export const Tabs = () => {};
+import React from 'react';
+
+interface TabsProps {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+}
+
+interface Tab {
+  id: string;
+  title: string;
+  content: string;
+}
+
+export const Tabs: React.FC<TabsProps> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const handleTabClick = (tab: Tab) => {
+    if (tab.id !== selectedTab.id) {
+      onTabSelected(tab);
+    }
+  };
+
+  return (
+    <>
+      <h1 className="title">{`Selected tab is ${selectedTab.title}`}</h1>
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map(tab => (
+              // eslint-disable-next-line
+              <li
+                key={tab.id}
+                data-cy="Tab"
+                onClick={() => handleTabClick(tab)}
+                className={selectedTab === tab ? 'is-active' : undefined}
```

Comment:
undefined ?
Make this in another way wirhout using undef

Reply:
I dont know if that was what you meant